openedx/features/colaraz_features/signals.py

```diff
@@ -0,0 +1,36 @@
+"""
+Signals for colaraz features django app.
+"""
+from logging import getLogger
+
+from django.conf import settings
+from django.dispatch import receiver
+from django.db.models.signals import post_save
+
+from lms.djangoapps.verify_student.models import ManualVerification
+from student.models import UserProfile
+
+
+logger = getLogger(__name__)
+
+
+@receiver(post_save, sender=UserProfile)
+def generate_manual_verification_for_user(sender, instance, **kwargs):
+    """
+    Generate ManualVerification for the User (whose UserProfile instance has been created).
+    """
+    if not (settings.FEATURES.get('ENABLE_AUTOMATIC_ACCOUNT_VERIFICATION') and created):
+        if settings.FEATURES.get('ENABLE_AUTOMATIC_ACCOUNT_VERIFICATION'):
+            logger.info('ManualVerification for user: {} has already been created.'.format(instance.user.email))
```

Comment:
No need to log this. If a new user is not being created, just don't do anything.